### PR TITLE
Resolved #2057: LTC - ICF-DD earned income FIATer

### LINKED ISSUE #2057: New Script idea - HC Fiater for Earned Income for LTC- ICF-DD clients
Some  Long-term care clients that are in ICF-DD facilities active on health care works & receives earned income, however, they are allowed the following deductions of $80 special personal allowance from earned income & FICA taxes withheld, if a FIATER can be created it'll be helpful. 

Policy exist: HCPM 23.15.10

### PR DESCRIPTION
**Pull request by IlseFerris**
BLIP: New script will FIAT earned income disregards on STAT/WKEX for LTC clients in ICF-DD's that receive earned income. Deductions will be FIATed for a selected month through the active budget period.

### CODE REVIEW

File: Script Files/ACTIONS/ACTIONS - LTC - ICF-DD DEDUCTION FIATER.vbs [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+	
+	'For an unknown (as of 06/24/2013) reason, some cases seem to stay in the first budget month and not move on. This is a fix for that.
+ 	EMReadScreen ending_bdgt_month, 5, 6, 14
+ 	If starting_bdgt_month = ending_bdgt_month then transmit
+	
+ 	'Resets the variables to check on the next month.
+ 	LBUD_check = ""
+NEXT
+
+script_end_procedure("Sucess, the budget has been updated to reflect earned income disregards. Please review the case prior to approval. Use NOTES - LTC - MA APPROVAL to case note the approval.")
```
Comment (theVKC) [inline Script Files/ACTIONS/ACTIONS - LTC - ICF-DD DEDUCTION FIATER.vbs]: "Success" is how this is spelled. This matters for stats collection. :)

**Comment by theVKC** [comment]
Please correct spelling error @IlseFerris. Otherwise, looks good.

**Comment by IlseFerris** [comment]
Done and done...ugh!

File: Script Files/ACTIONS/ACTIONS - LTC - ICF-DD DEDUCTION FIATER.vbs [hunk 2]
```diff
@@ -0,0 +1,306 @@
+'GATHERING STATS============================================================================================================
+name_of_script = "ACTIONS - LTC-ICF-DD DEDUCTION FIATER.vbs"
```
Comment (theVKC) [inline Script Files/ACTIONS/ACTIONS - LTC - ICF-DD DEDUCTION FIATER.vbs]: Ooooh... there needs to be a space between the dashes after LTC. It should match the file name exactly.

**Comment by theVKC** [comment]
Found one more issue @IlseFerris.

**Comment by IlseFerris** [comment]
Ok, let's try this again. I pull request thee!

**Comment by theVKC** [comment]
It's still not right, @IlseFerris... look at the name of the actual file and make sure it matches exactly.

**Comment by IlseFerris** [comment]
Sorry about that. Should be good to go now. Tested on the master and everything.

**Comment by theVKC** [comment]
Thank you!

**Pull request merged**